static/app/views/settings/projectPlugins/projectPlugins.tsx

```diff
@@ -42,16 +42,16 @@ class ProjectPlugins extends Component<Props> {
     const params = {orgId: organization.slug, projectId: project.slug};
 
     return (
-      <Panel>
-        <PanelHeader>
-          <div>{t('Legacy Integration')}</div>
-          <div>{t('Enabled')}</div>
-        </PanelHeader>
-        <PanelBody>
-          <PanelAlert type="warning">
-            <Access access={['org:integrations']}>
-              {({hasAccess}) =>
-                hasAccess
+      <Access access={['org:integrations']} project={project}>
+        {({hasAccess}) => (
+          <Panel>
+            <PanelHeader>
+              <div>{t('Legacy Integration')}</div>
+              <div />
```

Comment:
```suggestion
              <div>{t('Enabled')}</div>
```
need this for the table

Reply:
I removed it because other tables/pages don't label the column for the toggles.